src/content/release-notes.mdx

```diff
@@ -5,6 +5,46 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.31.0
+
+4 April 2025
+
+This version is compatible with Kubernetes versions 1.29 to 1.32 \
+Okteto Chart release 1.31 is designed to work with [Okteto CLI 3.6.x](https://github.com/okteto/okteto/releases/tag/3.6.0)
+
+### Deprecation Notice {#deprecation-notice-1.31}
+
+- ⚠️ Important: **Support for Docker Image Manifest Schema 1 images is removed in this** (1.32) due to upstream dependency changes. If you are using older images, they may fail to pull or deploy.\
+[Learn how to check and update your images →](self-hosted/manage/upgrade.mdx#upgrading-to-okteto-131x)
+
+### New Features {#new-features-1.31}
+
+- Added a new Public API endpoint to list users  <!-- 9175 -->
+- Added support for waking up Development Environments based on annotations in Docker Compose files  <!-- 9187 -->
+- Optimized the wake-up process to allow resources and Development Environments to start in parallel
+- Added support for Kubernetes 1.32 (previous support for 1.29 was removed)
+
+### Improvements {#improvements-1.31}
+
+- Namespace wake-up process will now respect Docker Compose `depends_on` with a feature flag
+- Improved error messages when ssh-keyscan error fails  <!-- https://github.com/okteto/app/pull/9197 -->
+- Patched IngressNightmare CVE-2025-1974 to enhance platform security  <!-- https://github.com/okteto/app/pull/9205 -->
+- Upgraded our BuildKit client to [0.20.2](https://github.com/moby/buildkit/releases/tag/v0.20.2)
+
+### Bug Fixes {#bug-fixes-1.31}
+
+- The `baggage.divert` header is now properly propagated to downstream services to indicate when a request is being diverted
+- Fixed an issue where logs appeared out of order on initial load  <!-- https://github.com/okteto/app/pull/9139 -->
+- Support bundles now include Ingress NGINX logs and Helm values for installations managed via ArgoCD
+- Updated Helm chart to include Okteto CLI 3.5.1, which resolves issues with Smart Builds  <!-- https://github.com/okteto/app/pull/9209 -->
```

Comment:
This doesn't apply as this version includes CLI `3.6.x`, which means that includes the Smart Builds fix. As this was already solve on a previous version, we don't include it in the next onex